packages/canvas/canvas-extract/src/CanvasExtract.ts

```diff
@@ -1,6 +1,9 @@
 import { RenderTexture } from '@pixi/core';
 import { CanvasRenderTarget } from '@pixi/utils';
 import { Rectangle } from '@pixi/math';
+import type { CanvasRenderer } from '@pixi/canvas-renderer';
+import type { DisplayObject } from '@pixi/display';
+import type { BaseRenderTexture } from '@pixi/core';
```

Comment:
Can you make sure these dependencies are including in package.json under dependencies?